spec/requests/admin/themes_controller_spec.rb

```diff
@@ -403,6 +403,42 @@
         expect(json["theme"]["auto_update"]).to eq(false)
         expect(UserHistory.where(action: UserHistory.actions[:change_theme]).count).to eq(1)
       end
+
+      context "with registered user_guardian_can_create_theme" do
+        it "doesn't allow importing a theme if the modifier returns false" do
+          plugin_instance = Plugin::Instance.new
+          i = 0
+          plugin_instance.register_modifier(
+            :user_guardian_can_create_theme,
+          ) do |val, admin_user, guardian|
+            expect(admin_user).to eq(admin)
+            expect(guardian.user).to eq(admin)
+            i += 1
+            i % 2 == 1
```

Comment:
Instead of using a count which is harder to follow, I think we can just use a boolean variable and then update it based on the response which we want to assert for.